apisix/plugins/traffic-split.lua

```diff
@@ -247,9 +244,15 @@ function _M.access(conf, ctx)
         return
     end
 
-    local weighted_upstreams, match_flag
+    local weighted_upstreams
+    local match_flag = true
+
     for _, rule in ipairs(conf.rules) do
-        match_flag = true
+        if not rule.match then
```

Comment:
Better to add descriptions to document that when match is absent， the rule is unconditional.

Reply:
Updated.